src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotification.java

```diff
@@ -99,25 +99,27 @@ private Map<String, String> applicant1TemplateVars(final CaseData caseData, fina
 
     private Map<String, String> applicant2TemplateVars(final CaseData caseData, final Long id) {
         Map<String, String> templateVars = commonContent.mainTemplateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1());
-        templateVars.putAll(courtDocumentDetails(caseData, templateVars.get(APPLICATION_REFERENCE), templateVars.get(COURT_EMAIL)));
+        templateVars.putAll(courtDocumentDetails(caseData));
         templateVars.putAll(serveAnotherWayTemplateVars(false, caseData));
         return templateVars;
     }
 
     private Map<String, String> serveAnotherWayTemplateVars(boolean soleServingAnotherWay, CaseData caseData) {
         Map<String, String> templateVars = new HashMap<>();
 
+        boolean servedAnotherWay = soleServingAnotherWay && caseData.isDivorce();
+        String divorceOrDissolution = caseData.isDivorce() ? "divorce" : "";
```

Comment:
Does this get adjusted for the Welsh versions?

Reply:
Added code to get Welsh text for Welsh language preference